go.mod

```diff
@@ -4,5 +4,6 @@ go 1.13
 
 require (
 	github.com/gorilla/websocket v1.4.2
+	github.com/joho/godotenv v1.3.0
```

Comment:
You add them to go.mod and they become necessary dependency for everyone who uses the library

Reply:
Oh yeah, of course. Added it actually just for local testing but yeah, I'm still too accustomed to just do `git add .` sadly. :D